core/whistle/src/api/wapi_notifications.erl

```diff
@@ -445,6 +448,23 @@
                                         ]).
 -define(DENIED_EMERGENCY_BRIDGE_TYPES, []).
 
+%% Customer update
+-define(CUSTOMER_UPDATE_HEADERS, [<<"Account-ID">>]).
+-define(OPTIONAL_CUSTOMER_UPDATE_HEADERS, [<<"Recipient-ID">>
+                                           ,<<"User-Type">>
+                                           ,<<"Subject">>
+                                           ,<<"From">>
+                                           ,<<"Reply-To">>
+                                           ,<<"To">>,<<"CC">>,<<"BCC">>
+                                           ,<<"HTML">>
+                                           ,<<"Plain">>
```

Comment:
Please look at the `-define(...)` of `DEFAULT_OPTIONAL_HEADERS`; you have duplicated most of them here, yet included the defaults below. I'd just remove them from this listing and let the macro include them for you!